qemu/tests/sr_iov_hotplug.py

```diff
@@ -170,7 +212,7 @@ def _find_pci():
 
             # Assign static IP to the hotplugged interface
             if params.get("assign_static_ip", "no") == "yes":
-                cmd = []
+                cmd = "service networking restart"
```

Comment:
this cmd initiation is only for ubuntu, right?
I'm thinking that if ubuntu is not a main stream of testing, should we initiate cmd as "service network restart" to reduce the times of modifying from "service networking restart" to "service network restart"?